notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -0,0 +1,604 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "190e8e4c-461f-4521-ae7f-3491fa827ab7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Automatic Device Selection with OpenVINO™\n",
+    "\n",
+    "The [Auto device](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_AUTO.html) (or AUTO in short) selects the most suitable device for inference by considering the model precision, power efficiency and processing capability of the available [compute devices](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_Supported_Devices.html). The model precision (i.e. FP32, FP16, INT8, etc.) is the first consideration to filter out the devices that cannot run the network efficiently.\n",
+    "\n",
+    "Next, if dedicated accelerators are available, these devices are preferred (e.g. integrated and discrete [GPU](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_GPU.html#doxid-openvino-docs-o-v-u-g-supported-plugins-g-p-u) or [VPU](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_VPU.html)). [CPU](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_CPU.html) is used as the default “fallback device”. Please note that AUTO makes this selection only once at the model load time. \n",
+    "\n",
+    "When using accelerator device like GPUs, loading models to these devices may take a long time. To address this challenge for applications that require fast first inference response, AUTO starts inferencing immediately on the CPU and then transparently shifts inferencing to the GPU once it is ready. This dramatically reduces the time to first inference.\n",
+    "\n",
+    "\n",
+    "![Auto Device Selection logic](https://user-images.githubusercontent.com/15709723/161451847-759e2bdb-70bc-463d-9818-400c0ccf3c16.png \"Auto Device Selection\")\n"
```

Comment:
```suggestion
    "<center><img src="https://user-images.githubusercontent.com/15709723/161451847-759e2bdb-70bc-463d-9818-400c0ccf3c16.png" width=580 alt="Auto Device Selection logic" /></center>\n"
```
The png becomes too large. I suggest using a different method, like this ^

Reply:
Please make this change and then it is good to merge. There are a few minor copy/formatting changes, but I will make those later.